src/components/Link.react.js

```diff
@@ -50,10 +50,10 @@ export default class Link extends Component {
     render() {
         const {className, style, id, href} = this.props;
         /*
-            * ideally, we would use cloneElement however
-            * that doesn't work with dash's recursive
-            * renderTree implementation for some reason
-            */
+         * ideally, we would use cloneElement however
+         * that doesn't work with dash's recursive
+         * renderTree implementation for some reason
+         */
```

Comment:
Sorry for this noise - it seems `Prettier` still picks up some formatting issues here and there from time to time. I think the formatting tests in Cirlce are maybe not 100% right, as it seems to miss these formatting issues.